src/main/scala/temple/generate/kube/KubernetesGenerator.scala

```diff
@@ -91,7 +91,7 @@ object KubernetesGenerator {
       service.dbImage,
       name,
       ports = Seq(),
-      env = service.envVars.map(x => EnvVar(x._1, x._2)),
+      env = service.dbEnvVars.map(x => EnvVar(x._1, x._2)),
```

Comment:
Why doesn’t Service already contain EnvVars rather than tuples?

Reply:
The way I saw it - the `kube.Spec.EnvVar` here have a certain JSON encoding defined on them that's pretty specific to KubeGen, whereas the `OrchestrationType.Service` above should be generic to all orchestration

Reply:
Ok, fair enough. Worth noting that `service.dbEnvVars.map(EnvVar.tupled)` is equivalent, and is arguably nicer than listing each component with `_i`